oslc4j-client-bundle/pom.xml

```diff
@@ -0,0 +1,103 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.lyo.clients</groupId>
+  <artifactId>oslc4j-client-bundle</artifactId>
+  <version>2.4.0-SNAPSHOT</version>
+  <packaging>bundle</packaging>
+
+
+<name>${bundle.symbolicName}</name>
+  <url>http://maven.apache.org</url>
+
+  <properties>
+    <bundle.symbolicName>oslc4j-client-bundle</bundle.symbolicName>
+    <bundle.namespace>org.oslc4j.client.bundle</bundle.namespace>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>osgi_R4_core</artifactId>
+      <version>1.0</version>
+      <type>jar</type>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>osgi_R4_compendium</artifactId>
+      <version>1.0</version>
+      <type>jar</type>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
+  
+    <build>
+    <plugins>
+      <plugin>
+        <!-- | the following instructions build a simple set of public/private
+          classes into an OSGi bundle -->
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.5.0</version>
+        <extensions>true</extensions>
+        <configuration>
+          <manifestLocation>META-INF</manifestLocation>
+          <instructions>
+            <Bundle-SymbolicName>${bundle.symbolicName}</Bundle-SymbolicName>
+            <Bundle-Version>${project.version}</Bundle-Version>
+		            <_exportcontents>
+		    org.eclipse.lyo.client.exception,
+			org.eclipse.lyo.client.oslc,
+			org.eclipse.lyo.client.oslc.jazz,
+			org.eclipse.lyo.client.oslcresources,
+            </_exportcontents>
+            <Import-Package>
+              javax.servlet;version="2.5.0",
```

Comment:
I think the POM file for the client was quite big. Are you sure it will work with such a small number of packages imported once someone tries to invoke it?